src/compas/geometry/primitives/plane.py

```diff
@@ -298,3 +298,16 @@ def transform(self, T):
         """
         self.point.transform(T)
         self.normal.transform(T)
+
+    def offset(self, distance):
+        """Returns a new offset plane by a given distance.
+        Plane normal is used as positive direction.
+
+        Parameters:
+        -----------
+        plane: compas.geometry.Plane
+
```

Comment:
there can't be blank lines in the parameter list. the parameters after the blank line will not be recognized...